process/block/preprocess/rewardsHandler.go

```diff
@@ -73,6 +70,11 @@ func NewRewardTxHandler(
 	if rewardTxPool == nil || rewardTxPool.IsInterfaceNil() {
 		return nil, process.ErrNilRewardTxDataPool
 	}
+	if economicsRewards == nil {
```

Comment:
Maybe you can also create IsInterfaceNil method and check in the same way as the other interfaces are checked here for nil

Reply:
Done.